enterprise/ha/src/main/java/org/neo4j/kernel/ha/HaSettings.java

```diff
@@ -101,7 +101,15 @@
         @Description("Round robin")
         round_robin,
 
-        @Description("Fixed")
-        fixed
```

Comment:
Wait, what? We can't remove "fixed"... that would break existing customer config, wouldn't it?